README.md

```diff
@@ -114,52 +114,58 @@ steps:
   run: dotnet pack --configuration Release <my project>
 - name: Publish the package to GPR
   run: dotnet nuget push <my project>/bin/Release/*.nupkg
+```
 
-# Authenticates packages to push to Azure Artifacts
-- uses: actions/setup-dotnet@v2
+### Azure Artifacts
+```yml
+- uses: actions/setup-dotnet@v3
   with:
     source-url: https://pkgs.dev.azure.com/<your-organization>/_packaging/<your-feed-name>/nuget/v3/index.json
   env:
     NUGET_AUTH_TOKEN: ${{secrets.AZURE_DEVOPS_PAT}} # Note, create a secret with this name in Settings
 - name: Publish the package to Azure Artifacts
   run: dotnet nuget push <my project>/bin/Release/*.nupkg
+```
 
-# Authenticates packages to push to nuget.org.
-# It's only the way to push a package to nuget.org feed for macOS/Linux machines due to API key config store limitations.
-- uses: actions/setup-dotnet@v2
+### nuget.org
+```yml
```

Comment:
Do we need to delete hint about authentication ?

Reply:
It just looks like it's deleted, but it's in line 140 under the `>**Note**` tag.